go.sum

```diff
@@ -1184,8 +1184,8 @@ github.com/iotexproject/iotex-antenna-go/v2 v2.5.1/go.mod h1:8pDZcM45M0gY6jm3PoM
 github.com/iotexproject/iotex-election v0.3.5-0.20210611041425-20ddf674363d h1:/j1xCAC9YiG/8UKqYvycS/v3ddVsb1G7AMyLXOjeYI0=
 github.com/iotexproject/iotex-election v0.3.5-0.20210611041425-20ddf674363d/go.mod h1:GRWevxtqQ4gPMrd7Qxhr29/7aTgvjiTp+rFI9KMMZEo=
 github.com/iotexproject/iotex-proto v0.5.0/go.mod h1:Xg6REkv+nTZN+OC22xXIQuqKdTWWHwOAJEXCoMpDwtI=
-github.com/iotexproject/iotex-proto v0.6.3 h1:wVBnI+oaT+rcw8AbD7xRZ+mNaqsWPQn6SlRbShRiUQ8=
-github.com/iotexproject/iotex-proto v0.6.3/go.mod h1:Hxct427jhlu08PHQ44Jwh//hZVhXKq+XO/Mdqwc6dNc=
+github.com/iotexproject/iotex-proto v0.6.4-0.20240820095805-53b815d06d26 h1:waJKULV/hZbWlsI+GabUXlchSUHwaePSCZWUGNPyVNI=
+github.com/iotexproject/iotex-proto v0.6.4-0.20240820095805-53b815d06d26/go.mod h1:Hxct427jhlu08PHQ44Jwh//hZVhXKq+XO/Mdqwc6dNc=
```

Comment:
can you file a PR in proto repo?

Reply:
https://github.com/iotexproject/iotex-proto/pull/158